aip/general/0126.md

```diff
@@ -55,12 +55,16 @@ message Book {
   - An exception to this rule is if there is a clearly useful zero value. In
     particular, if an enum needs to present an `UNKNOWN`, it is usually clearer
     and more useful for it to be a zero value rather than having both.
+- The other values **should not** be prefixed by the name of the enum itself.
+  This generally requires users to write `MyState.MYSTATE_ACTIVE` in their
+  code, which is unnecessarily verbose.
+  - Note that some languages (including C++) hoist enum values into the parent
+    namespace, which can result in conflicts for enums with the same values. To
+    avoid this, multiple enums in the same namespace **must not** share any
+    values.
```

Comment:
Do we want to be explicit in saying that to avoid this, APIs **may** prefix enum values with the name of the enum, but should do so consistently within the enum? (I appreciate there's already an implicit "may" by the parent bullet being "should not" but I think it would make it clearer.)

Reply:
Done.